pymc3/distributions/transforms.py

```diff
@@ -50,6 +50,16 @@ def jacobian_det(self, x):
         return tt.log(tt.abs_(grad))
 
 
+class VectorTransform(Transform):
+
+    def jacobian_det(self, x):
+        f_inv = self.backward(x)
+        J, _ = theano.scan(lambda i, f, x: tt.grad(f[i], x),
```

Comment:
why hardcode tt.grad function? We may know closed form solution without scan

Reply:
Interesting insight is that naive theano.gradient.jacobian works a bit slower than this implementation